README.md

```diff
@@ -40,6 +40,7 @@ current editor using the keymap `ctrl-shift-m`.
     ````bash
     $ apm install mathjax-wrapper
     ````
+  4. Optionally you may use pandoc to render the Markdown preview. Please have a look at the pandoc section below troubleshooting.
```

Comment:
I removed the troubleshooting section in Galadirith/markdown-preview-plus@5d764120d19c9453fe6fd8319c40a5a37d6b02d1. Could you please remove the reference to it.